cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+type args struct {
+	ThriftFile string `positional-arg-name:"thrift-file"`
+}
+
+var opts struct {
+	Relative string `long:"relative" description:"Output paths will be relative to this directory"`
+	Args     args   `positional-args:"yes" required:"yes"`
+}
+
+func do(relative string, inputThriftFile string, output io.Writer) error {
+	module, err := compile.Compile(inputThriftFile)
+	if err != nil {
+		return err
+	}
+
+	for _, included := range module.Includes {
```

Comment:
This will only do direct dependencies. Don't we want to list every Thrift file, both direct and transitive?

You can use [`module.Walk`](https://godoc.org/go.uber.org/thriftrw/compile#Module.Walk) to do the work for you.